qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -104,8 +127,7 @@ class DbExperimentDataV1(DbExperimentData):
     version = 1
     verbose = True  # Whether to print messages to the standard output.
     _metadata_version = 1
-    _executor = futures.ThreadPoolExecutor()
-    """Threads used for asynchronous processing."""
+    _job_executor = futures.ThreadPoolExecutor()
```

Comment:
Why this is class variable while one for analysis is instance variable?

Reply:
As a class variable this is shared by all experiment data classes. The other (analysis) one is local to a single experiment data class because we want to restrict it to a single-worker so that it only runs a single callback at a time. This is so callbacks can be cancelled since a running future cant be cancelled. Since jobs have a separate method for cancelling this isn't needed for them and they can all be running simultaneously.

Reply:
Make sense. Thanks.